app/buildpc.jsx

```
@@ -0,0 +1,1881 @@
+                      {option.priceDiff ? (
+                        <Text style={{
+                          color: selectedVariant === option ? '#fff' : '#888',
+                          fontSize: 12,
+                          marginLeft: 6,
+                          fontWeight: '600'
+                        }}>
+                          +{option.priceDiff.toLocaleString('vi-VN')}₫
+                        </Text>
+                      ) : null}
+                      {selectedVariant === option && (
+                        <Ionicons
```

ESLint

[error] 'Ionicons' is not defined.

(react/jsx-no-undef)